buildlib/tools/check_tls_perf_caps.sh

```diff
@@ -15,6 +15,8 @@ LD_LIBRARY_PATH=${UCX_PR_PATH}/lib "${UCX_PR_PATH}"/bin/ucx_info -v
 LD_LIBRARY_PATH=${UCX_LEGACY_PATH}/lib old_out=$("${UCX_LEGACY_PATH}"/bin/ucx_info -d)
 LD_LIBRARY_PATH=${UCX_PR_PATH}/lib new_out=$("${UCX_PR_PATH}"/bin/ucx_info -d)
 
+export UCX_TCP_BRIDGE_ENABLE=y #bridge devices are not shown by default since v1.16
```

Comment:
why would we want to run on bridge device?

Reply:
it can try running on ephemeral docker network devices and result in failures

Reply:
to be able to compare perf settings with UCX v1.15 (there is no such var there and default behavior is yes)

Reply:
we do not run anything here, just execute `ucx_info -d`, so I think it is safe

Reply:
ok